extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -844,6 +845,141 @@ describes.realWin(
       });
     });
 
+    describe('exposes api', () => {
+      let ampConsent;
+      let consentElement;
+
+      beforeEach(() => {
+        toggleExperiment(win, 'tcf-post-message-proxy-api', true);
```

Comment:
Wouldn't toggling the experiments be better in `beforeAll` and `afterAll`?

Reply:
I'm not sure we have that functionality built in?